.gitignore

```diff
@@ -0,0 +1,4 @@
+.Rproj.user
+.Rhistory
+.RData
+.Ruserdata
```

Comment:
`.gitignore` files tell git which files to ignore when updating your repository. Here you want to add stuff that is very specific to a user or their environment.

For example `.Rhistory` keeps a history of what you were doing in your RStudio session, so you wouldnt want that stuff floating around between you / mira etc.